easybuild/easyconfigs/p/PCL/PCL-1.7.2-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'PCL'
+version = '1.7.2'
+versionsuffix = '-Python-2.7.11'
+
+homepage = 'http://pointclouds.org/'
+description = """The Point Cloud Library (PCL) is a standalone, large scale, open project for 2D/3D image and
+ point cloud processing."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/PointCloudLibrary/pcl/archive/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [('CMake', '3.5.2')]
+dependencies = [
+    ('Boost', '1.60.0', versionsuffix),
+    ('Eigen', '3.2.8'),
```

Comment:
```
grep -R 'Eigen' $EASYBUILD_PREFIX/software/PCL/1.7.2-intel-2016a-Python-2.7.11/include | grep '#include' | wc -l
53
```

example:

```
.../software/PCL/1.7.2-intel-2016a-Python-2.7.11/include/pcl-1.7/pcl/point_cloud.h:#include <Eigen/StdVector>
.../software/PCL/1.7.2-intel-2016a-Python-2.7.11/include/pcl-1.7/pcl/point_cloud.h:#include <Eigen/Geometry>
```

@wpoely86: To me, this is a reason to make `Eigen` a runtime dep.